.unreleased/pr_7257

```diff
@@ -0,0 +1 @@
+Fixes: #7257 Force English in the pg_config command executed by cmake to avoid unexpected building errors
```

Comment:
You can add a "Thanks" entry with your GitHub account to make public your contribution.